### ISSUE TITLE
Map GROUP value not saving on map switch

### ISSUE THREAD

**Issue by ghost** [yenatch/crowdmap#20]
Short video re-creating the issue: https://i.imgur.com/d8AOX8R.gifv

When the map GROUP value is changed (in this situation, to alter the roof tile appearance in a town) from within Map Properties, you click 'save' but it reverts to the old GROUP value upon returning to the map. A different value such as MUSIC does save on exit and does not revert when returning to the map.

**Comment by yenatch** [comment]
yeah. i should probably grey it out so you can't edit it, since it doesn't save this way. changing the group would also mean some reordering the same way adding a map does it, which it doesn't do yet

**Comment by SachaSCG** [comment]
But changing the group also makes the design of the roofs change, how can this be done alternatively then?

**Comment by ghost** [comment]
> But changing the group also makes the design of the roofs change, how can this be done alternatively then?

That's a good point - goldenrod and azelea town share the same tileset, but the roof graphics are completely different. I wonder how it is coded to do that, do you know yenatch?

**Comment by yenatch** [comment]
you can change the group for real by moving the map into the right group in `constants/map_constants.asm` and `maps/map_headers.asm`. i think it's just those two files.
